styles.css

```diff
@@ -0,0 +1,9 @@
+#image {
+  display: block;
+
+  max-width: 100%;
+
+  margin-left: auto;
+
+  margin-right: auto;
```

Comment:
I think it would be better not to put new lines between each property, because in the standard of good writing practice or when you use linters it will not pass